src/index.js

```diff
@@ -0,0 +1,97 @@
+import './style.css';
+
+import { dragStart, dragOver, drop } from './drag.js';
+import {
+  addNewTask, editTask, deleteTask, clearAllCompleted,
+} from './add_and_remove.js';
+import updateStatus from './status.js';
+
+const taskList = document.getElementById('task-list');
+
+let allTasks = [
+];
+
+function populate(tasks) {
+  taskList.innerHTML = '';
+
+  tasks.forEach((task) => {
+    const listItem = document.createElement('li');
+    listItem.draggable = true;
+    listItem.classList.add('border-bottom', 'p-3', 'pt-2', 'pb-2', 'd-flex', 'flex-row', 'justify-content-between', 'align-items-center');
+    listItem.id = task.index;
+
+    const spanItem = document.createElement('span');
```

Comment:
Please use **ES6 syntax** as it will make your code cleaner and it has more concise syntax and it does not have any of its own `this` .